Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Build:
+#   $ docker build -t yottadb:latest .
```

Comment:
As discussed, can you add a link to the README file for instructions on general docker setup.

Reply:
Need to mention (either in the README or in this Dockerfile) the minimum version of Docker that is needed for this to work (17.05 I think you said).